srcStatic/Log.cpp

```diff
@@ -11,55 +11,55 @@ namespace {
 	// Static (constant and zero) initialization happens before dynamic initialization
 	// Such values are well defined if accessed by a global object constructor before Main/DllMain starts
 	// This pointer is safely set to null before other globals are initialized
-	LogDestination* loggerError = nullptr;
-	LogDestination* loggerMessage = nullptr;
-	LogDestination* loggerDebug = nullptr;
+	LogDestination* logDestinationError = nullptr;
+	LogDestination* logDestinationMessage = nullptr;
+	LogDestination* logDestinationDebug = nullptr;
 }
 
 
 // Set logging output destination for each logging level
-// Caller owns the logger and is responsible for cleanup when logger is no longer required
-// Use `SetLoggerX(nullptr);` to unset a logger
+// Caller owns the log destination and is responsible for cleanup when log destination is no longer required
+// Use `SetLogDestinationXXXX(nullptr);` to unset a log destination
```

Comment:
Something about all the Xs seems kind of funny here. Might be my math background talking.

Reply:
I liked the multiple XXXXs look better than the single X previously, although it is really a minor thing to me (also not directly related to the other changes. If you prefer the single X I'll switch it back.

Reply:
I suppose it's a bit like `YYYY-MM-DD`. Of course those are fixed width fields, and we have a variable length name at the end. That might actually be what was bothering me about it. When I saw a specific number of Xs, I assumed the variable portion would be that length, but it's not.